sktime/transformations/series/fabba.py

```diff
@@ -0,0 +1,1253 @@
+
+        Parameters
+        ----------
+            start_value : float
+                starting value of the time series
+            symbol_sequence : char or int
+                symbolized time series in form of string
+
+        Returns
+        -------
+            ts : 1D ndarray
+                reconstructed time series
+        """
+        # Get pieces from symbols
+        pieces = self._custom_inverse_symbolize(symbol_sequence)
+
+        #
+        pieces = self._custom_quantize_lengths(pieces)
+
+        # Get back time series from inverse compress
+        ts = self._custom_inverse_compress(pieces, start_value)
+
+        return ts
+
+    def _inverse_transform(self, X, y=None):
+        """Inverse transform, inverse operation to transform.
+
+        private _inverse_transform containing core logic, called from inverse_transform
+
+        Parameters
+        ----------
+        X : Series, Panel, or Hierarchical data, of mtype X_inner_mtype
+            if X_inner_mtype is list, _inverse_transform must support all types in it
+            Data to be inverse transformed
+        y : Series, Panel, or Hierarchical data, of mtype y_inner_mtype, default=None
+            Additional data, e.g., labels for transformation
+
+        Returns
+        -------
+        inverse transformed version of X
+
+        Raises
+        ------
+        ValueError
+            If self.return_start_values is False
+        """
+        if self.return_start_values is False:
+            raise ValueError("Start values are needed for inverse transform")
+
+        num_series = len(X)
+        start_values = list()
+        symbol_sequences = list()
+
+        # Get symbol sequences and start values
+        for item in X:
+            if isinstance(item, pd.DataFrame):
+                # Flatten the single column to a 1D array
+                col_values = item.iloc[:, 0].values
+
+                # First value
+                start_values.append(col_values[0])
+
+                # Remaining values
+                symbol_sequences.append(col_values[1:])
+            else:
+                raise ValueError("X should be a list of DataFrames")
+
+        # Start parallel inverse transform
+        n_jobs = self.n_jobs
+
+        # Pool doesn't take -1, need to count cpu
+        if n_jobs==-1:
+            n_jobs = os.cpu_count()
+
+        p = Pool(n_jobs)
+
+        result = [p.apply_async(
+                func=self._inverse_transform_single_series,
+                args=(start_values[i], symbol_sequences[i],)
+                ) for i in range(num_series)]
+
+        p.close()
+        p.join()
+
+        # Get series fromr results
+        series = [res.get() for res in result]
+
+        # Return as a df-list
+        return_data = list()
+        for i in range(len(series)):
+            return_data.append(pd.DataFrame({"ts": series[i]}))
+
+        return return_data
+
+    def _get_fitted_params(self):
```

Comment:
I would recommend: instead of implementing this function, simply write the parameters to `self.cluster_centers_`, `self.num_clusters_`, etc. Then the default will export them.